core/arch/arm/plat-vexpress/conf.mk

```diff
@@ -29,7 +29,7 @@ CFG_HWSUPP_MEM_PERM_PXN := y
 CFG_WITH_STACK_CANARIES := y
 CFG_PM_STUBS := y
 CFG_GENERIC_BOOT := y
-CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= 1
+CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= n
```

Comment:
Do you mean `y` here? If `n` you could just drop the definition since it is the default value.